### DISCUSSION TITLE
What are the ProposalNamespaces that we can use when establishing a connection with V2?

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1555] [Developer Support]
We are integrating with walletconnect 2.0 and we'd like to know what are the ProposalNamespaces that we can use when establishing a connection. I know these
 eip155
 cosmos
 solana
 polkadot

But what are those for BNB beacon chain and THORChain, for example? Is there a list in the docs, perhaps? I've been searching for it. But have had no success 😦
We'd also appreciate knowing what are the supported methods and events per chain. Is there some documentation about this?

**Comment by GrahamMcBain** [comment]
WalletConnect uses CAIP namespaces.  See https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md

**Reply by xzilja** [reply]
This is question best suited to ask in https://github.com/WalletConnect/walletconnect-monorepo or even better, our support discord
